keyboards/dztech/dz60rgb/readme.md

```diff
@@ -8,14 +8,17 @@ A hotswap 60% RGB keyboard.
 * Hardware Supported: DZtech DZ60RGB V1 and V2
 * Hardware Availability: [KBDfans](https://kbdfans.com/)
 
-There are two versions of the DZ60RGB. Please use the appropriate firmware for your board.
+There are three versions of the DZ60RGB. Please use the appropriate firmware for your board.
 
 * V1: STM32F303 (Arm), takes .bin files
 * V2: ATmega32U4 (AVR), takes .hex files
+* V2.1: ATmega32U4 (AVR), takes .bin files [Flash Manual](https://docs.google.com/document/d/111qx6Qec4JqtIhWaZlMND-VuRnFtn9a-gJaHN8fsL7M/edit?usp=sharing)  
```

Comment:
Does KBDfans refer to this as rev2.1?

Reply:
No. They refer to it as "new" v2, perhaps to reserve v3 for yet another iteration.

Reply:
Ugh, alright then. :)